documentation/api/io.strimzi.api.kafka.model.EntityTopicOperatorSpec.adoc

```diff
@@ -60,7 +60,10 @@ spec:
       reconciliationIntervalSeconds: 60
       logging:
         type: external
-        name: customConfigMap
+        valueFrom:
+          configMapKeyRef:
+            name: customConfigMap
+            key: key
```

Comment:
Can you use in all these examples a better key example? E.g. `entity-operator-log4j.properties` etc.?